CONTRIBUTING.MD

```diff
@@ -0,0 +1,3 @@
+# Contributing
+We recommend using [IntelliJ](https://www.jetbrains.com/idea/download/#section=windows) as IDE for contributing. We also recommend installing the Gradle Plugin and the [Save Actions Plugin](https://plugins.jetbrains.com/plugin/7642-save-actions).
```

Comment:
can you add a sub-section `## IDE` because there will be more things to explain here.. And remove the `#section=windows` from the download link ;)